scripts/run_experiment.sh

```diff
@@ -38,29 +43,38 @@ cat << EOF > $GET_RESULTS_SH
 #SBATCH --time=00:00:59
 #SBATCH -A plgccbmc11-cpu
 
-SACCT_RESULT="\$(sacct -j $COLLECT_ID --format State,End)"
-echo \$SACCT_RESULT
-arrIN=(\${SACCT_RESULT//;/ })
-STATE=\$(echo \${arrIN[4]})
-if [[ "\$STATE" == "COMPLETED" ]]
-then
-    END=\$(echo \${arrIN[5]})
-    T_END=\$(date --date="\$END" +"%s%N")
-elif [[ "\$STATE" == "FAILED" ]]
-then
-    echo "Experiment failed, exiting..."
-    exit 1
-fi
+while true
+do
+    SACCT_RESULT="\$(sacct -j $COLLECT_ID --format State,End)"
+    echo \$SACCT_RESULT
+    arrIN=(\${SACCT_RESULT//;/ })
+    STATE=\$(echo \${arrIN[4]})
+    if [[ "\$STATE" == "COMPLETED" ]]
+    then
+        END=\$(echo \${arrIN[5]})
+        T_END=\$(date --date="\$END" +"%s%N")
 
-T_EXEC_SECS=\$(( (T_END - $T_START) / 1000000000 ))
-T_EXEC_NANS=\$(( (T_END - $T_START) % 1000000000 ))
+        echo $T_START
+        echo \$T_END
```

Comment:
Może by to oznaczyć?

```suggestion
        echo \"T_START = $T_START\"
        echo \"T_END   = \$T_END\"
```